app/src/main/java/com/example/android/sunshine/app/utils/PrefUtility.java

```diff
@@ -11,10 +11,34 @@
 
 public class PrefUtility {
 
+    private static final float DEFAULT_LATLONG = 0F;
```

Comment:
careful you are using 1 constant for two things. At first I thought it's LatLong a pair?